### DISCUSSION TITLE
Where does mirth writes logs for all the channels

### DISCUSSION THREAD

**Discussion by sal4455** [nextgenhealthcare/connect#5149] [Q&A]
Hi all, Where does mirth create logs for all the channels ,i.e. if channels with source connector as **"File Writer"** , transform all the messages ,
how can we know that some files got transformed and some are not  without checking in its UI, GUI and restapi ? 

My  question here is - "is there any log file where mirth writes all channel logs about messages got transformed or not ?"
I am using mirthconnect 3.11.0 and postgres database .

**Comment by pacmano1** [comment]
There is no such "one log to capture all".

**Comment by tonygermano** [comment]
You likely want to look into the Alerts section of Mirth.
